packages/hydrogen/package.json

```diff
@@ -10,22 +10,24 @@
     "directory": "packages/hydrogen"
   },
   "scripts": {
-    "build": "node ../../utils/build.mjs",
+    "build": "node ../../utils/build-builder.mjs",
     "test-e2e": "pnpm test test/test.js",
     "test": "jest --reporters=default --reporters=jest-junit --env node --verbose --bail --runInBand"
   },
   "files": [
     "dist",
     "edge-entry.js"
   ],
+  "dependencies": {
+    "ts-morph": "12.0.0"
```

Comment:
I think `@vercel/static-config` should also be here since it depends on `ts-morph`

Reply:
Let's continue discussion [here](https://github.com/vercel/vercel/pull/10479#discussion_r1320382820).